src/components/organisms/OakHeader/OakHeader.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { ReactElement, ReactNode } from "react";
+import styled, { css } from "styled-components";
+
+import {
+  OakFlex,
+  OakP,
+  OakHeading,
+  OakBox,
+  OakImage,
+  OakThemeProvider,
+  OakMaxWidth,
+} from "@/components/atoms";
+import { oakDefaultTheme } from "@/styles";
+
+export type OakHeaderProps = {
+  headingTitle: string;
+  heroImageSrc: string;
+  authorImageSrc: string;
+  authorName: string;
+  authorTitle: string;
+  subHeadingText: string;
+  breadcrumbs?: ReactElement;
+  children?: ReactNode;
+};
+
+const StyledAutorImage = styled(OakImage)`
+  width: 50px;
+  height: 50px;
+  margin-right: 12px;
+  img {
+    border-radius: 50%;
+  }
+`;
+
+const OakHeaderCss = css<OakHeaderProps>``;
+
+const UnstyledComponent = (props: OakHeaderProps) => {
+  const {
+    headingTitle,
+    breadcrumbs,
+    authorName,
+    authorTitle,
+    subHeadingText,
+    heroImageSrc,
+    authorImageSrc,
+  } = props;
+  return (
+    <OakThemeProvider theme={oakDefaultTheme}>
```

Comment:
I think the theme provider should only be at app or page level

Reply:
I can't get this to pass the snapshot test, it keeps saying the background theme is undefined when removed

Reply:
ah yeah you will need to provide the ...provider in the tests, either wrap the component to be rendered or you can update the render method to render with providers

Reply:
✅